airflow/providers/docker/operators/docker.py

```diff
@@ -169,6 +173,7 @@ def __init__(
         self.dns_search = dns_search
         self.docker_url = docker_url
         self.environment = environment or {}
+        self._private_environment = private_environment or {}
```

Comment:
```suggestion
        self.private_environment = private_environment or {}
```

Reply:
@feluelle  I make this an underscore attribute so that it doesn't get rendered on the frontend at all.

Reply:
Ah I understand.